<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterScreen</class>
 <widget class="QDialog" name="RegisterScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>461</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: rgb(70, 153, 123);
    font-size: 20px;
    font-weight: 600;
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Roboto&quot;, &quot;Helvetica Neue&quot;, Arial, sans-serif;
    qproperty-alignment: AlignHCenter;
}
</string>
   </property>
   <property name="text">
    <string>Visa and Airline Reservation System</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 13.5px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Enter Username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 13.5px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;;
    font-size: 13.5px;
    font-weight: 600;
	color: rgb(0, 0, 0);
}
</string>
   </property>
   <property name="text">
    <string>Confirm Password:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;
}

/* Hover effect */
QLineEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QLineEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;
}

/* Hover effect */
QLineEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QLineEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>170</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    /* Typography */
    font-family: &quot;Segoe UI&quot;, &quot;Inter&quot;, &quot;Arial&quot;;
    font-size: 14px;
    font-weight: 400;

    /* Text &amp; Background */
    color: #1f1f1f;
    background-color: #ffffff;

    /* Border */
    border: 1px solid #cfd6e0;
    border-radius: 6px;

}

/* Hover effect */
QLineEdit:hover {
    border-color: #9bbce8;
    background-color: #fcfdff;
}

/* Focus effect */
QLineEdit:focus {
    border-color: #1b6ca8;
    background-color: #ffffff;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #6fdc8c,
        stop:1 #2fa866
    );
    color: white;
    border: none;
    font-size: 15px;
    font-weight: 600;
}

QPushButton:hover {
    background: qlineargradient(
        x1:0, y1:0, x2:0, y2:1,
        stop:0 #7fe09a,
        stop:1 #34b673
    );
}

QPushButton:pressed {
    background: #2fa866;
}
</string>
   </property>
   <property name="text">
    <string>Register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    
	background-color: rgb(82, 82, 82);
	color: rgb(255, 255, 255);
    border: none;
    font-size: 15px;
    font-weight: 600;
}


</string>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
